services/QuillLMS/client/app/bundles/Teacher/components/classrooms/reauthorize_google_modal.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+interface ReauthorizeGoogleModalProps {
+  googleLink: string;
+  close: () => void;
+}
+
+export default class ReauthorizeGoogleModal extends React.Component<ReauthorizeGoogleModalProps, {}> {
```

Comment:
I think this is probably based on an existing component, but since we're trying to move towards functional components in general, I'd rewrite this as:

```
const ReauthorizeGoogleModal = ({ googleLink, close}: ReauthorizeGoogleModalProps) => {
  function handleReauthorizeClick() {
    window.location.href = googleLink
  }

  return ... [everything you currently have in the return block of the render function]
}